requirements.txt

```diff
@@ -9,12 +9,13 @@ gitchangelog
 mock; python_version <= '2.7'
 pdbpp
 pre_commit
-psycopg2
```

Comment:
not installable on pypy but cffi variant is api compatible and seems to work everywhere. should be good enough for testing